src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { GridLayoutProp, applyGridClasses } from '../../grid/Grid/Grid'
+
+interface CardProps {
+  layout?: 'standardDefault' | 'flagDefault' | 'flagMediaRight'
+  headerFirst?: boolean
+  containerProps?: React.HTMLAttributes<HTMLDivElement>
+}
+
+export const Card = (
+  props: CardProps & React.HTMLAttributes<HTMLLIElement> & GridLayoutProp
+): React.ReactElement => {
+  const {
+    layout = 'standardDefault',
+    headerFirst,
+    children,
+    className,
+    gridLayout,
+    containerProps,
+    ...divProps
+  } = props
+
+  // TODO determine how to implement the tablet:grid-col-4 class properly, should this be li class?
+  const gridClasses = gridLayout && applyGridClasses(gridLayout)
+
+  const classes = classnames(
+    'usa-card',
+    {
+      'usa-card--header-first': headerFirst,
+      'usa-card--flag': layout === 'flagDefault' || layout === 'flagMediaRight',
+      'usa-card--media-right': layout === 'flagMediaRight',
+    },
+    gridClasses,
+    className
+  )
+
+  return (
+    <li className={classes} data-testid="Card" {...divProps}>
+      <div className="usa-card__container" {...containerProps}>
```

Comment:
There are cases in the test cards of this container also needing classes passed down. For example, the `border-ink` class is applied to some of the test cards.

Bug: The way it is implemented right now the `containerProps` className would override the existing `usa-card__container` className.

Reply:
How would you fix this bug? I don't think it makes sense to add a containerClass prop...